scripts/install.sh

```diff
@@ -3,22 +3,22 @@ set -e
 
 # Dafault inputs.
 KEEP_ETHEREUM_PASSWORD_DEFAULT="password"
-CONFIG_FILE_PATH_DEFAULT=$(realpath -m $(dirname $0)/../config.toml)
+CONFIG_DIR_PATH_DEFAULT=$(realpath -m $(dirname $0)/../configs)
```

Comment:
I would love it if we could avoid `realpath`—it relies on a newer version of bash than comes with macOS by default, so it requires folks to immediately install something else in the default environment we support :/

Rather than try to guess at what directory a script is invoked from (this is still error-prone, it turns out, because shells), thoughts on just assuming it's invoked as `scripts/install.sh` as the instructions note? I've moved a couple of other scripts to this approach, though they're more utility scripts than day-to-day dev scripts.

Reply:
I just found that `realpath` was [committed](https://github.com/coreutils/coreutils/commit/77ea441f79aa115f79b47d9c1fc9c0004c5c7111) 8 years ago as part of GNU coreutils and thinking that shells _should_ include it by now but I'm far from being an expert on what includes what :)

> thoughts on just assuming it's invoked as `scripts/install.sh` as the instructions note?

Sorry, I think I might need a bit of clarification here. Instead of:
```
CONFIG_DIR_PATH_DEFAULT=$(realpath -m $(dirname $0)/../configs)
```
should we replace it with:
```
CONFIG_DIR_PATH_DEFAULT="../configs"
```
?

If not, can you pls point to your script where you recently made a similar change?

Reply:
Yep, exactly that!

macOS ships an old version of bash due to a license shift (GPL v2 -> v3), basically. The default shell for new users as of Catalina and going forward is zsh (partly) for this reason.

Reply:
I've changed the scripts, ready for another shot.